src/screens/ENSAssignRecordsSheet.js

```diff
@@ -109,14 +113,22 @@ export default function ENSAssignRecordsSheet() {
   );
 
   const [hasSeenExplainSheet, setHasSeenExplainSheet] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      setHasSeenExplainSheet(Boolean(await getSeenOnchainDataDisclaimer()));
```

Comment:
can we move this method one line up in case of a failure? inside a try catch probably, i'm seeing errors locally, probably because i don't have this value so this is failing

Reply:
Hmm, I don't think this throws anyway?

<img width="808" alt="Screen Shot 2022-04-08 at 12 18 46 pm" src="https://user-images.githubusercontent.com/7336481/162350261-800e18ef-9ee7-4bfd-a083-819f0a84b134.png">

Reply:
ah right